addons/arsenal/functions/fnc_refresh.sqf

```diff
@@ -53,6 +54,11 @@ if (_updateVirtualItems) then {
 // Don't refresh left panel if in loadout tab
 if (!isNull findDisplay IDD_loadouts_display) exitWith {};
 
+if (!_animate) then {
+    GVAR(refreshing) = true;
+    [{GVAR(refreshing) = false}, nil, 3] call CBA_fnc_execAfterNFrames;
```

Comment:
execAfterNFrames won't work in 3den, but I guess it won't ever be reached so it's ok?
ref https://github.com/acemod/ACE3/pull/9542/files#r1369643874

Reply:
Maybe on line 38 we add `if (is3den) exitWith {};`?